src/shared/models/constants.ts

```diff
@@ -31,6 +31,8 @@ export const EMPTY_SUBMIT_DRAFT_QA_TEAM_ERROR_DESCRIPTION = 'Select a team for I
 export const EMPTY_SUBMIT_PRIVATE_MINISTER_TEAM_ERROR_DESCRIPTION = 'Select a team for Private Office and Minister sign off stages.';
 export const EMPTY_SUBMIT_TOPIC_ERROR_DESCRIPTION = 'Select a topic before submitting.';
 export const EMPTY_SUBMIT_TOPIC_ERROR_TITLE = 'No topic selected';
+export const EMPTY_SUBMIT_NOMINATED_CONTACT_DESCRIPTION = 'Please add at least one nominated contact before submitting.';
```

Comment:
```suggestion
export const EMPTY_SUBMIT_NOMINATED_CONTACT_DESCRIPTION = 'Add at least one nominated contact before submitting.';
```

[Guidance](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) is not to say "please", as it implies a choice when there isn't one.